src/Redux/missions/missionsSlice.js

```diff
@@ -0,0 +1,77 @@
+import {
+  createAsyncThunk,
+  createSlice,
+  isRejectedWithValue,
+} from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const initialState = {
+  missions: [],
+  status: 'idle',
+  error: null,
+  loading: false,
+  reservedMission: [],
+};
+
+const URL = 'https://api.spacexdata.com/v3/missions';
+
+export const fetchMissions = createAsyncThunk(
+  'missions/fetchMissions',
+  async () => {
+    try {
+      const response = await axios.get(URL);
+      return response.data;
```

Comment:
- [ ] Kindly consider dispatching an action  to store the selected data  which are 

```
* mission_id
* mission_name
* description
```

in Redux store
doing so will help you to meet the below requirement
<img width="737" alt="Screen Shot 2023-05-19 at 14 16 35" src="https://github.com/Kwaku28/Space-Travelers-Hub/assets/60146030/2173ec9d-91e0-4ad3-83cf-1e556f35e10e">